examples/command/portals/kafka/warpstream/application_team/run_ockam.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+set -ex
+
+if [[ $OCKAM_DEVELOPER != 'true' ]]; then
+    echo "Set OCKAM_DEVELOPER"
```

Comment:
why do we need this?

Reply:
We actually don't. I was debugging if the environment variable is seen in the docker compose. It's removed now, thanks.